### PR TITLE
PR for to fix calculateFee() issue when getting fees

### PR DESCRIPTION
**Pull request by jarenal**
I've taken advantage for to remove the unused parameter 'block' in calculateFee()

### CODE REVIEW

**Review by vekexasia — COMMENTED** [review]
Hmm. Not sure this is ok. I think block is being passed and should be used to calculate fees using system.

**Comment by jarenal** [comment]
Hi @vekexasia,

In the source code I found a TODO tag saying that the block parameter is unused and should be removed, look at here:

https://github.com/RiseVision/rise-node/blob/development/logic/block.js#L448

Regards.

**Comment by vekexasia** [comment]
I need you to double check if calculateFee is called anywhere with the block parameter if I remind correctly that method had a "fee" paremeter which was meant to be removed. Maybe the comment is just a leftover of that `fee` param.

**Comment by vekexasia** [comment]
@jarenal please remove the function entirely. I've checked and it's not used anywhere. Probably a leftover from lisk old codebase.

File: logic/block.js [hunk 1]
```diff
@@ -444,10 +444,9 @@ Block.prototype.getHash = function (block) {
  * Returns send fees from constants.
  * @param {block} block
  * @return {number} fee
- * @todo delete unused input parameter
  */
-Block.prototype.calculateFee = function (block) {
-	return constants.fees.send;
+Block.prototype.calculateFee = function () {
```
Comment (vekexasia) [inline logic/block.js]: remove this function entirely. Not used anywhere.

**Review by vekexasia — CHANGES_REQUESTED** [review]

File: logic/block.js (hunk 1 above)
Comment (jarenal) [reply logic/block.js]: Done!

**Review by vekexasia — APPROVED** [review]

**Pull request merged**